data/tlg0007/tlg082a/tlg0007.tlg082a.perseus-grc4.xml

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt><title>Τὰ παλαιὰ τῶν Λακεδαινίων ἐπιτηδεύματα</title>
+<author>Plutarch</author>
+<editor n="Teubner" role="editor">Grēgorios N. Vernardakēs</editor>
+<sponsor>Perseus Project, Tufts University</sponsor>
+<principal>Gregory Crane</principal>
+<respStmt>
+<resp>Prepared under the supervision of</resp>
+<name>Lisa Cerrato</name>
+<name>Rashmi Singhal</name>
+<name>Bridget Almas</name>
+</respStmt>
+<funder n="org:NEH">The National Endowment for the Humanities</funder>
+</titleStmt>
+<publicationStmt>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>Medford, MA</pubPlace>
+<authority>Perseus Project</authority>
+<date type="release">2010-12-13</date></publicationStmt>
+<sourceDesc>
+<biblStruct>
+<monogr>
+<author>Plutarch</author>
+<title xml:lang="lat">Moralia</title>
+<editor role="editor">Gregorius N. Bernardakis</editor>
+<imprint>
+<pubPlace>Leipzig</pubPlace>
+<publisher>Teubner</publisher>
+<date>1889</date>
+</imprint>
+<biblScope unit="volume">2</biblScope>
+</monogr>
+<ref target="https://archive.org/details/moralia07plut/page/180">The Internet Archive</ref></biblStruct>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+<editorialDecl>
+<correction method="silent">
+<p>optical character recognition</p>
+</correction>
+</editorialDecl>
+<p>Text encoded in accordance with the latest EpiDoc standards</p><p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture</p><refsDecl n="CTS">
+<!--<state unit="section" />-->
+<cRefPattern matchPattern="(\w+)" n="section" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"><p>This pointer pattern extracts sections</p></cRefPattern></refsDecl>
+<!--<refsDecl doctype="TEI.2">
+<state n="chunk" unit="stephpage" />
+</refsDecl>-->
+</encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="grc">Greek</language>
+<language ident="eng">English</language>
+</langUsage>
+</profileDesc>
+<revisionDesc>
+<change who="RS" when="2010-05">tagged and parsed</change>
+<change who="Sophia Elzie" when="2019-06-28">EpiDoc and CTS Conversion</change>
+</revisionDesc>
+</teiHeader>
+<text xml:lang="grc">
+<body>
+<pb xml:id="v.2.p.181" />
+<div n="urn:cts:greekLit:tlg0007.tlg082a.perseus-grc2" type="edition" xml:lang="grc"><milestone unit="section" />
```

Comment:
the file is failing (perhaps) because the name was not changed here so this needs to be grc4